interfaces/HO-Portal/src/app/program/phase-navigation/phase-navigation.component.html

```diff
@@ -1,21 +1,21 @@
 <span
   *ngFor="let phase of programPhases"
-  class="phase-navigation"
+  class="active-page-button"
```

Comment:
The name of this (CSS-only)components is a bit confusing/misleading.

It is a "page-button" that _can_ have the state "active", but using the term "active" in its name is a bit confusing.